site/layouts/shortcodes/orange-supra.html

```diff
@@ -10,6 +14,44 @@
       <li class="nav-item"><a href="#" class="nav-link active" aria-current="page">Personal</a></li>
       <li class="nav-item"><a href="#" class="nav-link">Business</a></li>
     </ul>
-    {{- $input -}}
+{{- if eq $mode "languages" }}
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item"><a class="nav-link active" href="#" aria-label="English version">EN</a></li>
+      <li class="nav-item"><a class="nav-link" href="#" aria-label="Version française" lang="fr" hreflang="fr">FR</a></li>
+      <li class="nav-item"><a class="nav-link" href="#" aria-label="Version en español" lang="es" hreflang="es">ES</a></li>
+    </ul>
+{{- end }}
+{{- if eq $mode "actions" }}
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a href="#" class="nav-link nav-icon">
+          <svg fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+            <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#search"/>
```

Comment:
I'd use {{< param docs_version >}} instead of 5.1

Reply:
Yep. I change it for the badge component as well.